components/shoplist-card/index.js

```diff
@@ -1,23 +1,22 @@
 //TODO : change the image path
 import path from 'path';
+import Link from 'next/link';
 export const ShopCard = (props) => {
   const { product } = props;
   const handleAddProduct = (item) => {
-    console.log(event);
     props.add.addCartItem(item);
     props.add.addShopListItem(item);
   };
 
   const handleRemoveProduct = (item) => {
     props.del.delCartItem(item);
-    {
-      path.join('./assets', product.image);
-    }
     props.del.delShopListItem(item);
   };
   return (
     <div className="shop-card">
-      <img src={path.join('./assets', product.image)} alt={product.name} />
+      <Link href={props.link.href} as={props.link.as}>
+        <img src={path.join('./assets', product.image)} alt={product.name} />
```

Comment:
could you please re-check the first argument path

Reply:
yes that is correct, it is wrt to public dir